tmt/steps/prepare/brand_new_allmighty_install/providers/brew.py

```diff
@@ -0,0 +1,15 @@
+"""
+Brew Artifact Provider
+"""
+
+import tmt.log
+from tmt.steps.prepare.brand_new_allmighty_install.providers.koji import KojiProvider
+
+
+class BrewProvider(KojiProvider):
+    """
+    Brew builds are just a special case of Koji builds
+    """
```

Comment:
Does this not need to support a different `_parse_artifact_id`? To support `brew.build:`? I guess the formats are synced to what testing-farm provides? I did not look into that, some reference would be helpful.

Reply:
No reference yet, I think we don't have a one as TF accepts this via JSON API. So we need to invent something and document it, for no I tried to use `koji.build:12345` or `brew.task:54321` as placeholders.